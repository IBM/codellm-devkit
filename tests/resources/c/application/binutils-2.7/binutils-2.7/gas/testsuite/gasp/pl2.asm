

	.ALTERNATE

	! ok
	!! also ok

foo	MACRO	
	! you can see me
	!! but not me
	! you can see me
	!! but not me
	but this "SHOULD !!BE OK"
	ENDM

	foo


define 	MACRO	val1,val2
	DB 	val1	! this comment will show up 
	DB	val2	!! this on won't
	ENDM

	define	0,1


	END
	
