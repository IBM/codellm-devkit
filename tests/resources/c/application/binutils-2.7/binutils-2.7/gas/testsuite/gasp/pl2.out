!

!

!	.ALTERNATE
!

!	! ok
	! ok
!	!! also ok
	!! also ok
!

!foo	MACRO	
!	! you can see me
!	!! but not me
!	! you can see me
!	!! but not me
!	but this "SHOULD !!BE OK"
!	ENDM
!

!	foo
!	! you can see me
	! you can see me
!	
!	! you can see me
	! you can see me
!	
!	but this "SHOULD !!BE OK"
	but this "SHOULD !!BE OK"
!

!

!define 	MACRO	val1,val2
!	DB 	val1	! this comment will show up 
!	DB	val2	!! this on won't
!	ENDM
!

!	define	0,1
!	DB 	0	! this comment will show up 
	.byte	0! this comment will show up 
!	DB	1	
	.byte	1
!

!

!	END
